### PR TITLE
Fixes #909 speed up build

### LINKED ISSUE #909: Build time needs drastic improvement
Almost 20 min on each PR, or update to PR. This takes way too much of our resources. We need faster feedback from CI

What can we remove from the default appveyor build and run nightly

Obviously Coverage. What about vignettes? Should we generate this only on nighty?
Anything else

This should be done consistently across all our R packages but ospsuite-r and reporting engine are the most important one ATM

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: appveyor-nightly.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+image: Visual Studio 2019
+
+version: '{build}'
```
Comment (msevestre) [inline appveyor-nightly.yml:3]: This is a copy of our old appveyor.yml. Only one creating artefact. It will run every night

File: appveyor.yml [hunk 2]
```diff
@@ -18,11 +18,9 @@ install:
 environment:
   USE_RTOOLS: true
   NOT_CRAN: true
-  KEEP_VIGNETTES: true
+  KEEP_VIGNETTES: false
```
Comment (msevestre) [inline appveyor.yml:21]: This is the one that will run on each commit

File: appveyor.yml [hunk 3]
```diff
@@ -79,15 +61,6 @@ artifacts:
   - path: '*.Rcheck\**\*.Rout'
     name: Logs
 
-  - path: "ospsuite_*ubuntu18.tar.gz"
```
Comment (msevestre) [inline appveyor.yml:82]: no artefact here

File: appveyor.yml [hunk 4]
```diff
@@ -39,29 +37,13 @@ build_script:
...
-before_test:
-  - cinst pandoc
-  - ps: $env:Path += ";C:\Program Files (x86)\Pandoc\"
-  - pandoc -v
-
-on_success:
-  - Rscript -e "covr::codecov(function_exclusions = c('print.*'))"
-
 test_script:
   - travis-tool.sh run_tests
```
Comment (msevestre) [inline appveyor.yml:45]: I would kike to exclude pksim related tests here and focus on core functions

**Review by msevestre — COMMENTED** [review]
Going to merge this to trigger builds and see if this works

File: appveyor-3.6.yml [hunk 5]
```diff
@@ -49,36 +46,11 @@ before_test:
   - ps: $env:Path += ";C:\Program Files (x86)\Pandoc\"
   - pandoc -v
 
-on_success:
-  - Rscript -e "covr::codecov(function_exclusions = c('print.*'))"
-
 test_script:
   - travis-tool.sh run_tests
 
-after_test:
```
Comment (msevestre) [inline appveyor-3.6.yml:58]: @IndrajeetPatil This needs to be removed as well.

**Comment by Yuri05** [comment]
if we don't create artifacts in every build - will our permalinks to the artifact in the latest build still work?
(e.g. https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/ospsuite.zip )

**Comment by msevestre** [comment]
This will point to the nightly now. So you 'll get the latest one a day or we need to trigger by hand

**Comment by Yuri05** [comment]
ok. not sure how AppVeyor knows what is nightly, but as long as it works - I don't care :)

**Pull request merged**